modules/mobile/app/models/mobile/v0/adapters/disability_rating.rb

```diff
@@ -21,13 +21,15 @@ def individual_ratings(response)
           return [] unless response['individual_ratings']
 
           response['individual_ratings'].map do |rating|
-            Mobile::V0::IndividualRating.new(
-              decision: rating['decision'],
-              effective_date: parse_date(rating['effective_date']),
-              rating_percentage: rating['rating_percentage']&.to_i,
-              diagnostic_text: rating['diagnostic_type_name']
-            )
-          end
+            unless rating['rating_end_date'].presence
```

Comment:
I don't think you need the presence check. I'd suggest leaving it off or using `present?` instead. Or even invert it to the positive case `if rating['rating_end_date'].nil?`